### PR TITLE
Fix docstring

### PR DESCRIPTION
**Pull request by Yash-10**
Fixing docstrings in various files.

### CODE REVIEW

**Comment by Yash-10** [comment]
Although this was inspired from #1191, there were some others that could also be fixed.

Thanks!

**Comment by DhruvSondhi** [comment]
Thanks @Yash-10 for the PR ... but this has been addressed in #1201 ... Please check the same :)

**Comment by Yash-10** [comment]
Oh, thanks for pointing out. I realized it now that it was a rare case of PR collision! Although, I think this PR tries to fix some more docstrings :) 

Maybe this could be merged after #1201 ?

Thanks!

**Comment by astrojuanlu** [comment]
Thanks @Yash-10! I merged #1201 first, would you please rebase this one? 🙏🏽

File: src/poliastro/earth/sensors.py [hunk 1]
```diff
@@ -52,9 +54,9 @@ def min_and_max_ground_range(h, η_fov, η_center, R):
 def max_and_min_ground_range_with_specific_azimuth(
     h, η_fov, η_center, β, φ_nadir, λ_nadir, R
 ):
-    """
-    Calculates the difference in ground-range angles from the η_center angle and the latitude and longitude of the target
+    """Calculates the difference in ground-range angles from the η_center angle and the latitude and longitude of the target
     for a desired phase angle,:math:`{\beta}`, used to specify where the sensor is looking.
```
Comment (DhruvSondhi) [inline src/poliastro/earth/sensors.py:58]: This wont work :)

**Review by DhruvSondhi — CHANGES_REQUESTED** [review]

File: src/poliastro/earth/sensors.py (hunk 1 above)
Comment (DhruvSondhi) [reply src/poliastro/earth/sensors.py:58]: @Yash-10 can you please revert this back?

File: src/poliastro/earth/sensors.py [hunk 2]
```diff
@@ -67,14 +65,14 @@ def max_and_min_ground_range_with_specific_azimuth(
         Angle of the total area that a sensor can observe.
     η_center: ~astropy.units.Quantity
         Center boresight angle.
-    β : ~astropy.units.Quantity
```
Comment (DhruvSondhi) [inline src/poliastro/earth/sensors.py:70]: I don't understand this change 🤔

File: src/poliastro/earth/sensors.py [hunk 3]
```diff
@@ -95,6 +93,7 @@ def max_and_min_ground_range_with_specific_azimuth(
     -----
     For further information, please take a look at "Fundamentals of Astrodynamics and Applications", 4th ed (2013)"
     by David A. Vallado, pages 853-860.
+
```
Comment (DhruvSondhi) [inline src/poliastro/earth/sensors.py:96]: No need for this ... I have tested this change ... not required IMO

File: src/poliastro/earth/sensors.py [hunk 4]
```diff
@@ -29,8 +28,8 @@ def min_and_max_ground_range(h, η_fov, η_center, R):
     -----
     For further information, please take a look at "Fundamentals of Astrodynamics and Applications", 4th ed (2013)"
     by David A. Vallado, pages 853-860.
-    """
```
Comment (DhruvSondhi) [inline src/poliastro/earth/sensors.py:31]: same here not required change 🤔

**Review by DhruvSondhi — CHANGES_REQUESTED** [review]

File: src/poliastro/earth/sensors.py (hunk 1 above)
Comment (DhruvSondhi) [reply src/poliastro/earth/sensors.py:58]:
```suggestion
    for a desired phase angle, β, used to specify where the sensor is looking.
```

**Comment by Yash-10** [comment]
@astrojuanlu I have rebased it!

Thanks!

File: src/poliastro/earth/sensors.py (hunk 3 above)
Comment (Yash-10) [reply src/poliastro/earth/sensors.py:96]: This was done to preserve uniformity across all docstrings...So I think it is required..

File: src/poliastro/earth/sensors.py (hunk 4 above)
Comment (Yash-10) [reply src/poliastro/earth/sensors.py:31]: Same as above

File: src/poliastro/earth/sensors.py (hunk 1 above)
Comment (Yash-10) [reply src/poliastro/earth/sensors.py]: Thanks..

**Comment by Yash-10** [comment]
I attempted to solve some additional inconsistencies throughout the documentation and fixed a broken link in one of the docstrings.

@astrojuanlu Could you please review this whenever you have time?

Thanks!

File: src/poliastro/util.py [hunk 5]
```diff
@@ -30,14 +30,14 @@ def time_range(start, *, periods=50, spacing=None, end=None, format=None, scale=
     ----------
     periods : int, optional
         Number of periods, default to 50.
-    spacing : Time or Quantity, optional
+    spacing : ~astropy.time.Time or astropy.units.Quantity, optional
```
Comment (DhruvSondhi) [inline src/poliastro/util.py:33]: Please have consistency here as well 
```suggestion
    spacing : ~astropy.time.Time or ~astropy.units.Quantity, optional
```

**Review by DhruvSondhi — CHANGES_REQUESTED** [review]

File: src/poliastro/iod/izzo.py [hunk 6]
```diff
@@ -30,9 +30,9 @@ def lambert(k, r0, r, tof, M=0, numiter=35, rtol=1e-8):
     rtol : float, optional
         Relative tolerance of the algorithm, default to 1e-8.
 
-    Yields
-    ------
-    v0, v : tuple
+    Returns
```
Comment (astrojuanlu) [inline src/poliastro/iod/izzo.py:33]: This is incorrect, right? There's a `yield` below

File: src/poliastro/twobody/sampling.py [hunk 7]
```diff
@@ -18,7 +18,7 @@ def sample_closed(min_nu, ecc, max_nu=None, num_values=100):
     First sample the eccentric anomaly uniformly,
     then transform into true anomaly
     to minimize error in the apocenter,
-    see http://www.dtic.mil/dtic/tr/fulltext/u2/a605040.pdf
+    see https://apps.dtic.mil/dtic/tr/fulltext/u2/a605040.pdf
```
Comment (astrojuanlu) [inline src/poliastro/twobody/sampling.py:21]: Good catch!

**Review by astrojuanlu — COMMENTED** [review]
Except for one thing, I think this is good to go!

File: src/poliastro/twobody/orbit.py [hunk 8]
```diff
@@ -271,7 +271,7 @@ def from_coords(cls, attractor, coord, plane=Planes.EARTH_EQUATOR):
         ----------
         attractor: Body
             Main attractor
-        coord: astropy.coordinates.SkyCoord or BaseCoordinateFrame
+        coord: astropy.coordinates.SkyCoord or ~astropy.coordinates.BaseCoordinateFrame
```
Comment (DhruvSondhi) [inline src/poliastro/twobody/orbit.py:274]: Consistency issue here as well 
```suggestion
        coord: ~astropy.coordinates.SkyCoord or ~astropy.coordinates.BaseCoordinateFrame
```

**Review by DhruvSondhi — CHANGES_REQUESTED** [review]

File: src/poliastro/iod/izzo.py [hunk 9]
```diff
@@ -32,7 +32,7 @@ def lambert(k, r0, r, tof, M=0, numiter=35, rtol=1e-8):
 
     Yields
     ------
-    v0, v : tuple
+    v0, v : tuple (v0, v)
```
Comment (DhruvSondhi) [inline src/poliastro/iod/izzo.py:35]: Seems to be excessive ... already output is mentioned 🤔

File: src/poliastro/iod/izzo.py (hunk 6 above)
Comment (Yash-10) [reply src/poliastro/iod/izzo.py]: Thanks for pointing out!

**Review by astrojuanlu — APPROVED** [review]
LGTM!

**Pull request merged**